addons/google_calendar/utils/google_calendar.py

```diff
@@ -76,13 +76,15 @@ def insert(self, values, token=None, timeout=TIMEOUT):
 
     @requires_auth_token
     def patch(self, event_id, values, token=None, timeout=TIMEOUT):
-        url = "/calendar/v3/calendars/primary/events/%s?sendUpdates=all" % event_id
+        send_updates = self.google_service._context.get('send_updates', True)
+        url = "/calendar/v3/calendars/primary/events/%s?sendUpdates=%s" % (event_id, "all" if send_updates else "none")
```

Comment:
Utilizing the `sendUpdates=none` option leads to certain complications for participants who do not employ Google Calendar. For instance, it can generate duplicate events in the Outlook calendar. A more reliable approach is to apply `sendUpdates=externalOnly`, which ensures that emails are sent exclusively to non-Google calendars.